src/api.ts

```diff
@@ -188,13 +190,15 @@ function resolveVitestConfig(showWarning: boolean, configOrWorkspaceFile: vscode
   }
 
   if (vitest.pnp) {
-    // TODO: try to load vitest package version from pnp
+    const zipOpenFs = new ZipOpenFS({ libzip: getLibzipSync() })
+    const crossPlatformCompatFs = new PosixFS(zipOpenFs)
+    const version = JSON.parse(crossPlatformCompatFs.readFileSync(vitest.vitestPackageJsonPath, 'utf8')).version
```

Comment:
We cannot rely on the versions in the reference attribute from [packagelocator](https://yarnpkg.com/advanced/pnpapi#packagelocator) because the documentation states: "reference field should be considered an opaque string whose value may be **whatever** the PnP implementation decides to put there."

Thus, I followed the suggestion from [this link](https://yarnpkg.com/advanced/pnpapi#accessing-the-files) in the Yarn documentation to get the version directly from the library package.json.

Reply:
This is an insane amount of code that we need to load for everyone just to do this. Maybe instead we can do something like this: https://github.com/microsoft/playwright-vscode/blob/main/src/playwrightFinder.ts

Reply:
I could, in theory, get it directly from the project's `package.json`, but the version might not be exact (e.g., `>=2.0.0`), and there are cases such as version ranges and similar that will make it harder for us. I will check the suggested resolver to see if I can extract it from Vitest.

Reply:
I used `yarn exec` here as well since we are going to use it for the test execution, as discussed on the issue thread.